src/app/extensions/order-templates/shared/order-create-order-template/order-create-order-template.component.html

```diff
@@ -0,0 +1,9 @@
+<button type="submit" name="addProduct" [ngClass]="cssClass" (click)="openModal(modal)">
```

Comment:
type="button", remove the name - we don't need it and adapt also the 'basket-create-order-template' in the same way